projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -103,6 +103,17 @@ export function addToCart() {
     .click({ force: true });
 }
 
+export function waitForAddToCart() {
+  cy.route(
+    'GET',
+    `/rest/v2/electronics-spa/users/*/carts/*?fields=*&lang=en&curr=USD`
```

Comment:
like discussed, include the api_url and it will be the same as the other one, so we can use this function for both places